geomstats/_backend/pytorch/__init__.py

```diff
@@ -127,6 +124,14 @@ def repeat(a, repeats, axis=None):
     return torch.repeat_interleave(a, repeats, axis)
 
 
+def flip(x, axis):
```

Comment:
Remember to remove the previous `flip = _raise_not_implemented_error` at the top of the file.

Reply:
Yep, done in the next commit (normally) !